src/cnhp.cpp

```diff
@@ -10,19 +10,19 @@
 
 
 /*------------------------------- INCLUDES --------------------------------*/
-#include <cnglob.h>
+#include "cnglob.h"
 
-#include <ancrec.h>	// record: base class for rccn.h classes
-#include <rccn.h>	// HEATPLANT BOILER including mbr fcn decls NHPSTAGES HPSTAGESZ
-#include <msghans.h>	// MH_R1350
+#include "ancrec.h"	// record: base class for rccn.h classes
+#include "rccn.h"	// HEATPLANT BOILER including mbr fcn decls NHPSTAGES HPSTAGESZ
+#include "msghans.h"	// MH_R1350
 
-//#include <pp.h>   	//
```

Comment:
If these includes have been commented for years, I'd just take them out.

Reply:
Done, it is not being shown as outdated.

Reply:
Huh, that's so weird.